src/componets/test/test.tsx

```diff
@@ -0,0 +1,47 @@
+import React,{useState, useEffect, FunctionComponent} from "react";
+import axios from "axios";
+
+// type WeatherData = Function;
+type Employee = {
+  id?: number;
+  name?: string;
+};
+
+interface Props {
+    // children: ReactNode;
+    count: number;
+    connected: boolean;
+    type: string;
+}
+// const Counter:FunctionComponent<{ initial?: number }> = ({ initial = 0 }) => {
+const GetWeatherData:FunctionComponent<{
+    initial?: number;
+    WheaterData?: string;
+    data3?:Array<Employee>;
```

Comment:
what is data3 ? and Employee?

Reply:
testingan ko po pinag aaralan ko pa po yung tsx medyo  bibugbog ako ng tsx laguage .. pag hinapan na ako ng 'property type does not exist on type' hehehe medyo inaaway na ako ni TS

Reply:
done na po